Cargo.toml

```diff
@@ -7,6 +7,8 @@ authors = ["acolytec3"]
 categories = ["wasm"]
 readme = "README.md"
 edition = "2018"
+license = "(MIT OR Apache-2.0)"
```

Comment:
@holgerd77 Any concerns about this license?

Reply:
This was copied from the package.json btw likely better to pick one or the other imo

Reply:
Yeah, so no concerns or so, I have absolutely not insight about this choice, but if this was the license before we should rather just keep it and move on.

Changing the license for whatever reasons is always a rabbit whole.

(so, but to not cause confusion: I would say we can keep this addition to `Cargo.toml`)